src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/15.diff.sql

```diff
@@ -0,0 +1,98 @@
+﻿/*************************************************************
+    Background job checkpoint feature
+**************************************************************/
+
+/*************************************************************
+    Background job checkpoint table
+**************************************************************/
+
+IF NOT EXISTS (SELECT 1 FROM sys.tables WHERE name = 'BackgroundJobCheckpoint')
+BEGIN
+     CREATE TABLE dbo.BackgroundJobCheckpoint
+     (
+         Id bigint IDENTITY(1,1) NOT NULL,
```

Comment:
>Id bigint IDENTITY(1,1) NOT NULL, [](http://example.com/codeflow?start=9&length=33)

Is this column actually needed?

Reply:
Thanks for the review. It is not needed. Removed